acceptance/tests/direct_puppet/supports_utf8.rb

```diff
@@ -5,14 +5,20 @@
   app_type = File.basename(__FILE__, '.*')
   tmp_environment   = mk_tmp_environment(app_type)
 
-  filenames         = {'windows' => "C:/cygwin64/tmp/#{tmp_environment}_file"}
-  filenames.default = "/tmp/#{tmp_environment}_file"
+  tmp_file = {}
+  agents.each do |agent|
```

Comment:
i suppose i could.  but i'm pretty sure any way i do that will be a loop or a collection.  same same?
eg: `first_windows = agents.collect{ |a| a.platform =~ /windows/ }.first`